sdk/typescript/src/client/cryptography.ts

```diff
@@ -0,0 +1,358 @@
+import {
+	create_dkg_centralized_output,
+	create_imported_dwallet_centralized_step,
+	create_sign_centralized_output,
+	decrypt_user_share,
+	encrypt_secret_share,
+	generate_secp_cg_keypair_from_seed,
+	network_dkg_public_output_to_protocol_pp,
+} from '@dwallet-network/dwallet-mpc-wasm';
+import { bcs } from '@mysten/sui/bcs';
+import { decodeSuiPrivateKey } from '@mysten/sui/cryptography';
+import { Secp256k1Keypair } from '@mysten/sui/keypairs/secp256k1';
+import sha3 from 'js-sha3';
+
+import { IkaClient } from './ika-client';
+import { DWallet } from './types';
+import { UserShareEncrytionKeys } from './user-share-encryption-keys';
+import { encodeToASCII, u64ToBytesBigEndian } from './utils';
+
+/**
+ * Prepared data for the second round of Distributed Key Generation (DKG).
+ * Contains all cryptographic outputs needed to complete the DKG process.
+ */
+export type PreparedSecondRound = {
+	/** The centralized public key share along with its zero-knowledge proof */
+	centralizedPublicKeyShareAndProof: Uint8Array;
+	/** The centralized public output from the DKG process */
+	centralizedPublicOutput: Uint8Array;
+	/** The centralized secret key share (should be encrypted before storage) */
```

Comment:
clarify (in doc) please: this is the plaintext secret key share, not the encrypted one right?